sdks/python/apache_beam/io/gcp/datastore/v1/datastoreio.py

```diff
@@ -53,6 +53,9 @@
 
 class ReadFromDatastore(PTransform):
   """A ``PTransform`` for reading from Google Cloud Datastore.
+  
```

Comment:
We can drop this change. Web site changes are sufficient. The reasons this notice is not consistent, there are other IOs that will not work in streaming mode and they do not have comments. It is also likely to not get updated as the support is added.

Reply:
I still feel that we should add such information to code base as well.
But concerned of two independent sources of truth that can go out of sync.